docs/writing-policies/spec/host-capabilities/03-container-registry.md

```diff
@@ -79,3 +79,113 @@ the payload would be the following ones:
 * Input payload: `"busybox:latest"`
 * Output payload: `{ "digest": "sha256:69e70a79f2d41ab5d637de98c1e0b055206ba40a8145e7bddb55ccc04e13cf8f"}`
 
+## OCI Image index
```

Comment:
I would call it `Image manifest`. The Image index is a [type of manifest](https://github.com/opencontainers/image-spec/blob/main/image-index.md):

> The image index is a higher-level manifest which points to specific [image manifests](https://github.com/opencontainers/image-spec/blob/main/manifest.md), ideal for one or more platforms. While the use of an image index is OPTIONAL for image providers, image consumers SHOULD be prepared to process them.

There's no API to fetch the Image index from a registry, that's because you can obtain it by fetching the image manifest.

Because of that, I propose to replace all the `index` instances inside of this PR with `manifest`

Reply:
Maybe we can change the docs to call it `Image manifest` but making clear that the output can change depending of the OCI artifact manifest requested:

```console
$ crane manifest ghcr.io/kubewarden/policies/pod-privileged:latest | jq ""                                                                                                             
{
  "config": {
    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
    "mediaType": "application/vnd.wasm.config.v1+json",
    "size": 2
  },
  "layers": [
    {
      "digest": "sha256:6d803e1199b0c884741a43c6fbc2f1586839429f00681a3098116437d4210b71",
      "mediaType": "application/vnd.wasm.content.layer.v1+wasm",
      "size": 1285267
    }
  ],
  "schemaVersion": 2
}
```

Therefore, the output can be a `Image index` or a single `Image manifest`. And the policy must deal with it.

Reply:
@flavio  I've updated the docs changing the name and adding an statement that depending of the image the response can change. It can be a index manifest or a single image manifest.

Reply:
I"m happy with this approach, approving.